parquet-column/src/main/java/org/apache/parquet/io/api/Binary.java

```diff
@@ -48,6 +50,16 @@ private Binary() { }
 
   abstract public byte[] getBytes();
 
+  /**
+   * Variant of getBytes() that avoids copying backing data structure by returning
+   * backing byte[] of the Binary. Do not modify backing byte[] unless you know what
+   * you are doing.
+   * @return backing byte[] of correct size, with an offset of 0, if possible, else returns result of getBytes()
+   */
+  abstract public byte[] getBytesUnsafe();
```

Comment:
@julienledem you had some concern regarding "getBytesUnsafe" to which @rdblue posted an answer. Are you OK with this now?